index.js

```diff
@@ -52,11 +91,11 @@ app.get("/movies/Director/:Director", (req, res) => {
 });
 
 //adds movie to the list of favorites of a user
-app.post("/users/:id/movies/:id", (req, res) => {
```

Comment:
Express doesn't allow the use of to arguments with the same name `:id` in the URL . So I recommend updating this to have either the first or second argument(or both) to have a different name